<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Password Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#loginPage{
	background-image: url(:/images/images/background.jpg);
}

#formLoginWidget{ 
	background-color: rgba(255, 255, 255, 0.4);
	border-radius: 9px;
}

#errorlabel {
	color: red;
}

#input_password {
	border-radius: 7px;
}

#loginBtn {
	border-radius: 7px;
	background-color: rgba(255, 255, 255, 0.4);
}

#loginBtn:hover {
	background-color: rgba(255, 255, 255, 0.6);
}

#loginBtn:pressed {
	background-color: rgba(255, 255, 255, 0.8);
}

#welcomelabel {	
	color: rgba(255, 255, 255, 0.9)
}

#stackedWidget_2 {
	border: 2px solid rgb(255, 255, 255);
}

#viewWidget {
	background-color: rgb(245, 245, 245) 
}

#inputs_sideWidget {
	background-color: rgb(245, 245, 245) 
}

#email_input, #sitename_input, #username_input, #password_input {
	border-radius: 7px;
	border: 1px solid rgb(218, 218, 218);
}

#deleteBtn {
	background-color: red;
	border-radius: 7px;
	color: white;
	border: 0.5px solid black;
}

#deleteBtn:hover {
	background-color: rgb(236, 0, 0);
}

#deleteBtn:pressed {
	background-color: rgb(225, 0, 0);
}

#addBtn, #updateBtn {
	border-radius: 7px;
	border: 0.5px solid black;
}

#addBtn:hover, #updateBtn:hover {
	background-color: rgb(245, 245, 245);
}
#addBtn:pressed, #updateBtn:pressed {
	background-color: rgb(231, 231, 231)
}

#tableInfo {
	border:none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QWidget" name="formLoginWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QStackedWidget" name="stackedWidget_2">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="login_page">
              <widget class="QLineEdit" name="input_password">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>140</y>
                 <width>171</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Enter the password</string>
               </property>
              </widget>
              <widget class="QLabel" name="errorlabel">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>90</y>
                 <width>171</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
              <widget class="QPushButton" name="loginBtn">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>190</y>
                 <width>171</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
              <widget class="QLabel" name="welcomelabel">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>40</y>
                 <width>171</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Welcome back&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="homePage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="viewWidget" native="true">
          <widget class="QTableWidget" name="tableInfo">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>581</width>
             <height>581</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <column>
            <property name="text">
             <string>Site name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Username</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Password</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Email</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="inputs_sideWidget" native="true">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <widget class="QLineEdit" name="sitename_input">
           <property name="geometry">
            <rect>
             <x>32</x>
             <y>100</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Site name</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="username_input">
           <property name="geometry">
            <rect>
             <x>32</x>
             <y>140</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Username</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="password_input">
           <property name="geometry">
            <rect>
             <x>32</x>
             <y>180</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Password</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="email_input">
           <property name="geometry">
            <rect>
             <x>32</x>
             <y>220</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Email</string>
           </property>
          </widget>
          <widget class="QPushButton" name="addBtn">
           <property name="geometry">
            <rect>
             <x>32</x>
             <y>270</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
          <widget class="QPushButton" name="updateBtn">
           <property name="geometry">
            <rect>
             <x>32</x>
             <y>310</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
          <widget class="QPushButton" name="deleteBtn">
           <property name="geometry">
            <rect>
             <x>32</x>
             <y>430</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_developer"/>
   </widget>
   <addaction name="menuabout"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_developer">
   <property name="text">
    <string>About developer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
